tests/unit/conftest.py

```diff
@@ -202,3 +202,13 @@ def xtrigger_mgr() -> XtriggerManager:
             create_autospec(Scheduler, workflow=workflow_name, owner=user)
         )
     )
+
+@pytest.fixture(autouse=True)
+def prevent_symlinking(monkeypatch: pytest.MonkeyPatch, request):
+    if 'force_symlinks' not in request.keywords:
```

Comment:
Allows `test_install_workflow__symlink_target_exists` to opt out of this - it is actually testing the symlinking as such has it set up well for a test.